lib/driver.rb

```diff
@@ -0,0 +1,53 @@
+require_relative "csv_record"
+
+module RideShare
+  class Driver < CsvRecord
+    attr_reader :name, :vin, :trips
+    attr_accessor :status
+
+    def initialize(id:, name:, vin:, status: :AVAILABLE, trips: nil)
+      super(id)
+      @name = name
+      @vin = vin.length == 17 ? vin : (raise ArgumentError.new("Invalid Vin: #{vin.length} length"))
```

Comment:
Generally ternaries are only used for things where both branches are values.

I would do:
```suggestion
      if vin.length == 17
        @vin = vin
      else
        raise ArgumentError.new("Invalid Vin: #{vin.length} length")
      end
```
or:
```suggestion
      raise ArgumentError.new("Invalid Vin: #{vin.length} length" unless vin.length == 17
      @vin = vin
```